### DISCUSSION TITLE
How do I create animated textures?

### DISCUSSION THREAD

**Discussion by lvmdx** [MoreMcmeta/core#86] [Help Wanted]
Hello, I can't figure out what to do and how to do it. Where to throw your textures and how to make them animated

**Comment by soir20** [comment]
There's a [demo resource pack](https://www.curseforge.com/minecraft/texture-packs/moremcmeta-demo) that is a working example of how to configure the textures and where to put them. Let me know if there's something missing/unclear from the demo pack.
